### ISSUE TITLE
Unable to print larger font ("cannot read property width of undefined")

### ISSUE THREAD

**Issue by Favna** [oliver-moran/jimp#562]
# Expected Behavior

Font is printed to the image successfully

# Current Behavior

Jimp throws an error saying the width cannot be found despite the font file working in the past before v0.3.X. Oddly enough a smaller variant of the same bitmap font does work.

# Failure Information (for bugs)

```
TypeError: Cannot read property 'width' of undefined
    at drawCharacter (E:\UserFiles\DevProjects\ribbon\node_modules\jimp\dist\image-manipulation\text.js:69:12)
    at Jimp.printText (E:\UserFiles\DevProjects\ribbon\node_modules\jimp\dist\image-manipulation\text.js:93:5)
    at Jimp.print (E:\UserFiles\DevProjects\ribbon\node_modules\jimp\dist\image-manipulation\text.js:184:13)
    at TestCommand.run (E:\UserFiles\DevProjects\ribbon\src\commands\owner\test.js:45:14)
    at <anonymous>
```

## Steps to Reproduce

1. Download this larger font file: [fonts.zip](https://github.com/oliver-moran/jimp/files/2307060/fonts.zip)

    1. The smaller font that does work: [font-smaller.zip](https://github.com/oliver-moran/jimp/files/2307109/font-smaller.zip)

2. Write basic code for loading the font and printing it to the canvas:

```js
const Jimp = require('jimp');

async run() {
    try {
        const canvas = await Jimp.read(500, 150);
        const font = await Jimp.loadFont('path/to/font');

        canvas.print(font, 155, 10, 'test');

        return canvas.write('canvas.png');
    } catch (err) {
        return console.error(err);
    }
}

run();
```

3. Observe errors

## Context

-   Jimp Version: 0.3.4 and 0.3.5
-   Operating System: Windows 10 and Raspbian 9 Stretch
-   Node version: 8.11.2

## Failure Logs

See above at failure info

**Comment by Favna** [comment]
Update: I messed around a bit with BitMapFontGenerator, the program I use, and I managed to fix it with a new font but I honestly have no idea what the reason is why the bug occurs. The only difference is that I enabled spaces and small cap letters as well as the full alphabet while the previous one only had the letters I would ever need anyway. 

Fact still remains that pre-0.3.X my font worked just fine and post-0.3.X it doesn't.

The fixed font: [fixed-font.zip](https://github.com/oliver-moran/jimp/files/2307217/fixed-font.zip)
